main.cpp

```diff
@@ -241,35 +344,94 @@ struct CarWash //2)
     2)report errors
     3)transmit serial data back to computer
  */
-
+struct UsbInterface
+{
+    //1)speed in Mb per second
+    int usbSpeed = 40;
+    //2)standard (1.0 or 2.0 for USB ) 
+    int usbStandard = 1;
+    //3)cable length
+    float cableLength = 5; 
+    //4)connector type
+    char usbConnectorType = 'A';
+    //5)current it can receive for powering the board
+    float currentDrawForPower = 50;
```

Comment:
using 'usb' in the member variable names of a class named `UsbInterface` is redundant
- [ ] revise

Reply:
ok, removed word 'usb' from member variables.